packages/bbui/src/Form/Core/Picker.svelte

```diff
@@ -199,6 +201,8 @@
             aria-selected="true"
             tabindex="0"
             on:click={() => onSelectOption(getOptionValue(option, idx))}
+            on:mouseenter={e => onOptionMouseenter(e, option)}
+            on:mouseleave={e => onOptionMouseleave(e, option)}
```

Comment:
NAB: would be nice to dispatch custom `optionenter` and `optionleave` events here, which can be just proxied on up the component chain rather than having to have a bunch of prop drilling with NOPs but it's not a big deal - works the same.